packages/api/lambdas/db-indexer.js

```diff
@@ -113,7 +113,7 @@ function getParentId(type, record) {
   if (type === 'granule') {
     return record.collectionId;
   }
-  return null;
+  return undefined;
```

Comment:
Please update the jsdocs to indicate that this can return a string _or_ undefined.